validation/contracts/ValidationStatus.sol

```diff
@@ -0,0 +1,4 @@
+contract ValidationStatus {
```

Comment:
please use rest codes only in the library.  this belongs in the app

Reply:
This needs to be present, and will be extended by the app.